src/Microsoft.AspNetCore.Localization/project.json

```diff
@@ -1,6 +1,10 @@
 {
   "version": "1.0.0-*",
-  "description": "Middleware for automatically applying culture information to HTTP requests.",
+  "description": "ASP.NET Core middleware for automatically applying culture information to HTTP requests. Culture information can be specified in the HTTP header, query string, cookie, or custom source.",
```

Comment:
I certainly hope we never remove any of these ways - that would be a breaking change. And if we add them, the comment remains correct. The goal is for the description to be sufficient to understand what's in the package and why you'd care to install it. Or, similarly, to understand whether it's ok to remove a reference to the package.